packages/appkit/src/client.ts

```diff
@@ -87,6 +89,10 @@ export class AppKit {
     this.initControllers(options)
     this.initOrContinue()
     this.version = options.sdkVersion
+    // Check on the next thick because wagmiAdapter authConnector is not immediately available
+    setTimeout(() => {
+      this.checkExistingConnection()
+    }, 0)
```

Comment:
do you need to trigger this here? shouldn't you do after / during authConnector initialization?